README.md

```diff
@@ -1,22 +1,47 @@
 ## Signature
 Complete any delivery service App with this Signature widget.
```

Comment:
You can have something like "A signature pad for capturing signatures" instead of the paragraph